tests/unit/spec/base/mozilla-client.js

```diff
@@ -458,7 +458,7 @@ describe('mozilla-client.js', function() {
 
         it('should consider outdated if user version is slightly less than latest version and strict option is true', function() {
             test(true, false, '46.0.1').toBeFalsy();
-            test(true, false, '45.0').toBeFalsy();
+            test(true, false, '46.0').toBeFalsy();
```

Comment:
I think the test description here sums already sums it up about as elequently as another comment could accomplish?